packages/material-ui/src/index.js

```diff
@@ -400,3 +400,5 @@ export * from './Zoom';
 
 // createFilterOptions is exported from Autocomplete
 export { default as useAutocomplete } from './useAutocomplete';
+
+export { StylesProvider as StyledEngineProvider } from '@material-ui/styled-engine';
```

Comment:
I'd vote for `StyledEngineProvider` to avoid a collision in names...

Reply:
> I'd vote for StyledEngineProvider to avoid a collision in names...

I started my comment with this too, until I realized that the `styled` module is a lot more used and already conflicts. What should we do with it?

Reply:
Ah sorry, my bad, I misunderstood your comment. Currently, the "old" `StylesProvider` is exported from `@material-ui/styles`, not the core. I re-exported the new one by e different name, thinking that it is re-exported from `@material-ui/core`.

In this case, I agree, we can just export it by the same name from the core.